partest-suite/src/test/resources/scala/tools/partest/scalajs/2.12.0-M4/BuglistedTests.txt

```diff
@@ -0,0 +1,15 @@
+# The tests in this file should pass but have never passed so far
+# use scala.tools.partest.scalajs.testunknownonly to only run tests
+# which are neither in BuglistedTests.txt, WhitelistedTests.txt or
+# BlacklistedTests.txt
+
+# Bug #2309
+run/sammy_after_implicit_view.scala
+run/sammy_restrictions_LMF.scala
+run/sammy_vararg_cbn.scala
+run/t5568.scala
+run/t6318_primitives.scala
+
+# fails assert(x == y, x + " == " + y)
+# where (x: java.lang.Long = 5) and (y: scala.math.BigDecimal = 5)
+run/equality.scala
```

Comment:
Actually, `run/equality.scala` is broken on master. It started failing in the nightly of April 12.